diff -ruw m68k-atari-mint-gcc-gcc-13-mintelf.old/isl/config.sub m68k-atari-mint-gcc-gcc-13-mintelf/isl/config.sub
--- m68k-atari-mint-gcc-gcc-13-mintelf.old/isl/config.sub	2020-12-31 15:34:29.000000000 +0100
+++ m68k-atari-mint-gcc-gcc-13-mintelf/isl/config.sub	2023-08-12 14:08:59.359089058 +0200
@@ -232,7 +232,7 @@
 	-psos*)
 		os=-psos
 		;;
-	-mint | -mint[0-9]*)
+	-mint | -mintelf | -mint[0-9]*)
 		basic_machine=m68k-atari
 		os=-mint
 		;;
@@ -826,7 +826,7 @@
 	miniframe)
 		basic_machine=m68000-convergent
 		;;
-	*mint | -mint[0-9]* | *MiNT | *MiNT[0-9]*)
+	*mint | *mintelf | -mint[0-9]* | *MiNT | *MiNT[0-9]*)
 		basic_machine=m68k-atari
 		os=-mint
 		;;
@@ -1496,7 +1496,7 @@
 	-ose*)
 		os=-ose
 		;;
-	-*mint | -mint[0-9]* | -*MiNT | -MiNT[0-9]*)
+	-*mint | -*mintelf | -mint[0-9]* | -*MiNT | -MiNT[0-9]*)
 		os=-mint
 		;;
 	-zvmoe)
@@ -1779,7 +1779,7 @@
 			-mpw* | -macos*)
 				vendor=apple
 				;;
-			-*mint | -mint[0-9]* | -*MiNT | -MiNT[0-9]*)
+			-*mint | -*mintelf | -mint[0-9]* | -*MiNT | -MiNT[0-9]*)
 				vendor=atari
 				;;
 			-vos*)
diff -ruw m68k-atari-mint-gcc-gcc-13-mintelf.old/mpc/build-aux/config.sub m68k-atari-mint-gcc-gcc-13-mintelf/mpc/build-aux/config.sub
--- m68k-atari-mint-gcc-gcc-13-mintelf.old/mpc/build-aux/config.sub	2020-10-20 18:26:31.000000000 +0200
+++ m68k-atari-mint-gcc-gcc-13-mintelf/mpc/build-aux/config.sub	2023-08-12 14:08:59.359089058 +0200
@@ -805,7 +805,7 @@
 		cpu=m68000
 		vendor=convergent
 		;;
-	*mint | mint[0-9]* | *MiNT | *MiNT[0-9]*)
+	*mint | *mintelf | mint[0-9]* | *MiNT | *MiNT[0-9]*)
 		cpu=m68k
 		vendor=atari
 		os=mint
@@ -1481,7 +1481,7 @@
 	ose*)
 		os=ose
 		;;
-	*mint | mint[0-9]* | *MiNT | MiNT[0-9]*)
+	*mint | *mintelf | mint[0-9]* | *MiNT | MiNT[0-9]*)
 		os=mint
 		;;
 	zvmoe)
@@ -1772,7 +1772,7 @@
 			mpw* | macos*)
 				vendor=apple
 				;;
-			*mint | mint[0-9]* | *MiNT | MiNT[0-9]*)
+			*mint | *mintelf | mint[0-9]* | *MiNT | MiNT[0-9]*)
 				vendor=atari
 				;;
 			vos*)
diff -ruw m68k-atari-mint-gcc-gcc-13-mintelf.old/mpfr/config.sub m68k-atari-mint-gcc-gcc-13-mintelf/mpfr/config.sub
--- m68k-atari-mint-gcc-gcc-13-mintelf.old/mpfr/config.sub	2020-07-10 13:59:02.000000000 +0200
+++ m68k-atari-mint-gcc-gcc-13-mintelf/mpfr/config.sub	2023-08-12 14:08:59.362422392 +0200
@@ -232,7 +232,7 @@
 	-psos*)
 		os=-psos
 		;;
-	-mint | -mint[0-9]*)
+	-mint | -mintelf | -mint[0-9]*)
 		basic_machine=m68k-atari
 		os=-mint
 		;;
@@ -826,7 +826,7 @@
 	miniframe)
 		basic_machine=m68000-convergent
 		;;
-	*mint | -mint[0-9]* | *MiNT | *MiNT[0-9]*)
+	*mint | *mintelf | -mint[0-9]* | *MiNT | *MiNT[0-9]*)
 		basic_machine=m68k-atari
 		os=-mint
 		;;
@@ -1496,7 +1496,7 @@
 	-ose*)
 		os=-ose
 		;;
-	-*mint | -mint[0-9]* | -*MiNT | -MiNT[0-9]*)
+	-*mint | -*mintelf | -mint[0-9]* | -*MiNT | -MiNT[0-9]*)
 		os=-mint
 		;;
 	-zvmoe)
@@ -1779,7 +1779,7 @@
 			-mpw* | -macos*)
 				vendor=apple
 				;;
-			-*mint | -mint[0-9]* | -*MiNT | -MiNT[0-9]*)
+			-*mint | -*mintelf | -mint[0-9]* | -*MiNT | -MiNT[0-9]*)
 				vendor=atari
 				;;
 			-vos*)
