diff -ruw gmp-4.3.2.orig/configfsf.sub gmp-4.3.2/configfsf.sub
--- gmp-4.3.2.orig/configfsf.sub	2010-01-08 06:09:02.000000000 +1000
+++ gmp-4.3.2/configfsf.sub	2026-02-16 16:03:58.800079980 +1000
@@ -227,7 +227,10 @@
 		os=-psos
 		;;
 	-mint | -mint[0-9]*)
-		basic_machine=m68k-atari
+		case $basic_machine in
+			*-*) basic_machine=`echo $basic_machine | sed 's/-.*/-atari/'` ;;
+			*)   basic_machine=$basic_machine-atari ;;
+		esac
 		os=-mint
 		;;
 esac
