diff -ru gmp.orig/config.in gmp/config.in
--- gmp.orig/config.in	2017-08-17 11:28:54.995856912 +1000
+++ gmp/config.in	2017-08-17 11:29:09.012738014 +1000
@@ -162,6 +162,7 @@
 #undef HAVE_HOST_CPU_m68040
 #undef HAVE_HOST_CPU_m68060
 #undef HAVE_HOST_CPU_m68360
+#undef HAVE_HOST_CPU_5475
 #undef HAVE_HOST_CPU_powerpc604
 #undef HAVE_HOST_CPU_powerpc604e
 #undef HAVE_HOST_CPU_powerpc750
diff -ru gmp.orig/configure gmp/configure
--- gmp.orig/configure	2017-08-17 11:28:54.999190295 +1000
+++ gmp/configure	2017-08-17 11:29:09.019404783 +1000
@@ -4555,28 +4555,10 @@
 
 echo "include_mpn(\`m68k/m68k-defs.m4')" >> $gmp_tmpconfigm4i
 
-    gcc_cflags="$gcc_cflags $fomit_frame_pointer"
-    gcc_cflags_optlist="arch"
-
-    # gcc 2.7.2 knows -m68000, -m68020, -m68030, -m68040.
-    # gcc 2.95 adds -mcpu32, -m68060.
-    # FIXME: Maybe "-m68020 -mnobitfield" would suit cpu32 on 2.7.2.
-    #
-    case $host_cpu in
-    m68020)  gcc_cflags_arch="-m68020" ;;
-    m68030)  gcc_cflags_arch="-m68030" ;;
-    m68040)  gcc_cflags_arch="-m68040" ;;
-    m68060)  gcc_cflags_arch="-m68060 -m68000" ;;
-    m68360)  gcc_cflags_arch="-mcpu32 -m68000" ;;
-    *)       gcc_cflags_arch="-m68000" ;;
-    esac
-
-    # FIXME: m68k/mc68020 looks like it's ok for cpu32, but this wants to be
-    # tested.  Will need to introduce an m68k/cpu32 if m68k/mc68020 ever uses
-    # the bitfield instructions.
     case $host_cpu in
-    m680[234]0 | m68360)  path="m68k/mc68020 m68k" ;;
-    *)                      path="m68k" ;;
+    m68000 | m68060) path="m68k" ;;
+    m680[234]0) path="m68k/mc68020 m68k" ;;
+    # other m68k CPU -> assume incompatible instruction set
     esac
     ;;
 
diff -ru gmp.orig/gmp-impl.h gmp/gmp-impl.h
--- gmp.orig/gmp-impl.h	2017-08-17 11:28:54.999190295 +1000
+++ gmp/gmp-impl.h	2017-08-17 11:29:09.019404783 +1000
@@ -3448,7 +3448,7 @@
 
 /* As per glibc. */
 #if defined (__GNUC__) && ! defined (NO_ASM)                    \
-  && HAVE_HOST_CPU_FAMILY_m68k && GMP_LIMB_BITS == 32
+  && HAVE_HOST_CPU_FAMILY_m68k && ! HAVE_HOST_CPU_5475 && GMP_LIMB_BITS == 32
 #define BSWAP_LIMB(dst, src)						\
   do {									\
     mp_limb_t  __bswapl_src = (src);					\
