diff --git a/syscall/Makefile b/syscall/Makefile
index b382bc9..ff578af 100644
--- a/syscall/Makefile
+++ b/syscall/Makefile
@@ -10,6 +10,13 @@
 
 SHELL = /bin/sh
 
+OS = $(shell uname -s)
+ifeq ($(OS), Darwin)
+FLEXLIB = -ll
+else
+FLEXLIB = -lfl
+endif
+
 srcdir = .
 top_srcdir = ..
 subdir = syscall
@@ -47,7 +54,7 @@ gen-syscall0:
 	$(MAKE) gen-syscall CC="$(CC_FOR_BUILD)" CFLAGS="-O -Wall" COMPILE="$(CC_FOR_BUILD) -O -Wall"
 
 gen-syscall: $(gen_objs)
-	$(CC) $(gen_objs) -o $@ -lfl
+	$(CC) $(gen_objs) -o $@ $(FLEXLIB)
 
 %.c: %.y
 	bison -d --output=$@ $<
