diff -ru pml-2.03-old/pmlsrc/Makefile pml-2.03/pmlsrc/Makefile
--- pml-2.03-old/pmlsrc/Makefile	2013-11-21 11:23:11.379062834 +1000
+++ pml-2.03/pmlsrc/Makefile	2013-11-21 12:46:52.374630756 +1000
@@ -1,5 +1,5 @@
 CROSSDIR = /dsrg/bammi/cross-minix
-CROSSLIB = $(CROSSDIR)/lib
+CROSSLIB = $(CROSSDIR)/lib/$(LIB)
 CROSSBIN = $(CROSSDIR)/bin
 CROSSINC = $(CROSSDIR)/include
 WITH_SHORT_LIBS = 0
@@ -13,13 +13,13 @@
 all : $(ALL)
 
 libpml32.a :
-	make -f Makefile.32 clean
-	make -f Makefile.32 libpml32.a
+	$(MAKE) -f Makefile.32 clean
+	$(MAKE) -f Makefile.32 libpml32.a
 
 ifeq ($(WITH_SHORT_LIBS), 1)
 libpml.a :
-	make -f Makefile.16 clean
-	make -f Makefile.16 libpml.a
+	$(MAKE) -f Makefile.16 clean
+	$(MAKE) -f Makefile.16 libpml.a
 endif
 
 install : $(ALL)
@@ -33,7 +33,7 @@
 	cp math.h $(CROSSINC)
 
 clean:
-	make -f Makefile.32 clean
+	$(MAKE) -f Makefile.32 clean
 ifeq ($(WITH_SHORT_LIBS), 1)
-	make -f Makefile.16 clean
+	$(MAKE) -f Makefile.16 clean
 endif
diff -ru pml-2.03-old/pmlsrc/Makefile.16 pml-2.03/pmlsrc/Makefile.16
--- pml-2.03-old/pmlsrc/Makefile.16	2013-11-21 11:23:11.379062834 +1000
+++ pml-2.03/pmlsrc/Makefile.16	2013-11-21 12:46:21.117142611 +1000
@@ -16,13 +16,13 @@
 #	some machines support randomly ordered libraries, ordering
 #	them correctly doesn't hurt...
 CROSSDIR = /dsrg/bammi/cross-minix
-CROSSLIB = $(CROSSDIR)/lib
+CROSSLIB = $(CROSSDIR)/lib/$(LIB)
 CROSSBIN = $(CROSSDIR)/bin
 CROSSINC = $(CROSSDIR)/include
 
 AR = /dsrg/bammi/cross-gcc/bin/car
 CC = $(CROSSBIN)/mgcc
-CFLAGS = -mshort -O2 -fomit-frame-pointer -DIEEE -DNO_DBUG -I.
+CFLAGS = -mshort -O2 -fomit-frame-pointer $(CPU) -DIEEE -DNO_DBUG -I.
 
 LIB = $(CROSSLIB)
 
diff -ru pml-2.03-old/pmlsrc/Makefile.32 pml-2.03/pmlsrc/Makefile.32
--- pml-2.03-old/pmlsrc/Makefile.32	2013-11-21 11:23:11.379062834 +1000
+++ pml-2.03/pmlsrc/Makefile.32	2013-11-21 12:46:35.430655976 +1000
@@ -16,13 +16,13 @@
 #	some machines support randomly ordered libraries, ordering
 #	them correctly doesn't hurt...
 CROSSDIR = /dsrg/bammi/cross-minix
-CROSSLIB = $(CROSSDIR)/lib
+CROSSLIB = $(CROSSDIR)/lib/$(LIB)
 CROSSBIN = $(CROSSDIR)/bin
 CROSSINC = $(CROSSDIR)/include
 
 AR = /dsrg/bammi/cross-gcc/bin/car
 CC = $(CROSSBIN)/mgcc
-CFLAGS = -O2 -fomit-frame-pointer -DIEEE -DNO_DBUG -I.
+CFLAGS = -O2 -fomit-frame-pointer $(CPU) -DIEEE -DNO_DBUG -I.
 
 LIB = $(CROSSLIB)
 
